src/components/TimeCardPage/TimeSheet.tsx

```diff
@@ -61,21 +122,36 @@ export default function Page() {
         setCurrentTimesheetsToDisplay (userTimesheets, date); 
     }
     
-    const [userTimesheets, setUserTimesheets] = useState([]); 
-    const [currentTimesheets, setCurrentTimesheets] = useState([]);
-    const [selectedTimesheet, setTimesheet] = useState();
+    //TODO: default to first employee but idk if employee always exists
+    const [selected, setSelected] = useState<UserSchema>();
+    const [user, setUser] = useState<UserSchema>();
```

Comment:
Can you leave some documentation for what these state variables represent? Could get confusing